packages/core/tests/js/vite/many-plugins/run.ts

```diff
@@ -0,0 +1,17 @@
+import { imports, vite, type AstTypes } from '@sveltejs/cli-core/js';
+
+export function run(ast: AstTypes.Program): void {
+	const pMiddle = 'middlePlugin';
+	imports.addDefault(ast, { as: pMiddle, from: 'middle-plugin' });
+	vite.addPlugin(ast, { code: `${pMiddle}()` });
+
+	const pLast = 'lastPlugin';
+	imports.addDefault(ast, { as: pLast, from: 'last-plugin' });
+	[...Array(10)].forEach((_, i) => {
+		vite.addPlugin(ast, { code: `${pLast}(${i})` });
+	});
+
+	const pFirst = 'firstPlugin';
+	imports.addNamed(ast, { imports: { [pFirst]: pFirst }, from: 'first-plugin' });
+	vite.addPlugin(ast, { code: `${pFirst}()`, mode: 'prepend' });
```

Comment:
isn't this test actually testing the `prepend` `mode`? The test dosn't really have anything to do with many plugins, or am i missing something? Would suggest to only add 3-4 plugins overall and rename this test to be something like `prepend-mode`.

LMK if im missing something

Reply:
The idea was to have many plugins & prepend yes.
I'll do less plugins 👍